Documentation/GettingStarted.md

```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
 
 ```swift
 extension Reactive where Base: URLSession {
-    public func response(_ request: URLRequest) -> Observable<(Data, HTTPURLResponse)> {
+    public func response(request: URLRequest) -> Observable<(response: HTTPURLResponse, data: Data)> {
         return Observable.create { observer in
-            let task = self.dataTaskWithRequest(request) { (data, response, error) in
+
+            let task = self.base.dataTask(with: request) { (data, response, error) in
+            
                 guard let response = response, let data = data else {
-                    observer.on(.error(error ?? RxCocoaURLError.Unknown))
+                    observer.on(.error(error ?? RxCocoaURLError.unknown))
                     return
                 }
 
+
                 guard let httpResponse = response as? HTTPURLResponse else {
                     observer.on(.error(RxCocoaURLError.nonHTTPResponse(response: response)))
                     return
                 }
 
-                observer.on(.next(data, httpResponse))
+
+                observer.on(.next((httpResponse, data)))
                 observer.on(.completed)
             }
 
             task.resume()
 
-            return Disposables.create {
-                task.cancel()
-            }
+            return Disposables.create(with: task.cancel)
```

Comment:
It's better to leave the previous syntax - it's more commonly used and a matter of preference, not an actual fix.

Reply:
This line still needs to be reverted